docker-compose-test.yml

```diff
@@ -61,11 +63,11 @@ services:
   wiremock:
     image: wiremock/wiremock
     networks:
-      - hmpps_int
+      - hmpps
     container_name: inc-ui-wiremock
     restart: always
     ports:
       - '9091:8080'
 
 networks:
-  hmpps_int:
+  hmpps:
```

Comment:
there's no need to use a bespoke network for testing; the two docker-compose suites cannot be run concurrently anyway